### DISCUSSION TITLE
Any plans to support tensor parallelism ?

### DISCUSSION THREAD

**Discussion by vikigenius** [Lightning-AI/pytorch-lightning#17932] [DDP / multi-GPU / multi-node]
The strategy API kind of seems limited and I am not sure if it is even worth it to use PyTorch lightning if you want Tensor Parallelism.

Is there any plans for supporting Tensor Parallelism: https://pytorch.org/docs/stable/distributed.tensor.parallel.html

https://huggingface.co/transformers/v4.9.0/parallelism.html#tensor-parallelism

**Comment by altosaar** [comment]
Yes, this would be helpful to understand!
